### PR TITLE
Implemented feedback publishing by administrators

### PR DESCRIPTION
**Pull request by nikolapantelic-ftn**
- implemented new service methods for updaing feedbacks
- created buttons on front-end

### CODE REVIEW

**Comment by nikolapantelic-ftn** [comment]
[[E3.4] As a system administrator I want to publish feedbacks so that I could create a good image for the hospital. [1]](https://trello.com/c/xlmVlcVw/20-e34-as-a-system-administrator-i-want-to-publish-feedbacks-so-that-i-could-create-a-good-image-for-the-hospital-1)

**Review by rammba — CHANGES_REQUESTED** [review]
Synchronize this branch with master, and resolve conflicts first.

File: HospitalWebApp/Controllers/FeedbackController.cs [hunk 1]
```diff
@@ -52,11 +70,36 @@ public IActionResult Create(UserFeedbackDto userFeedbackDto)
...
+            {
+                return Ok("Feedback successfully published.");
+            }
+            else
+            {
+                return BadRequest();
+            }
+        }
+
+        private static void UpdateUserFeedbackDate(UserFeedback userFeedback)
```
Comment (rammba) [inline HospitalWebApp/Controllers/FeedbackController.cs:94]: This method was removed before and date is set in userFeedbackDtoToUserFeedback method.

**Review by rammba — CHANGES_REQUESTED** [review]

File: HospitalWebApp/Controllers/FeedbackController.cs (hunk 1 above)
Comment (nikolapantelic-ftn) [reply HospitalWebApp/Controllers/FeedbackController.cs]: Must have missed it while fixing merge conflicts. Removed.

**Review by rammba — APPROVED** [review]

**Review by BoKna031 — APPROVED** [review]

**Comment by rammba** [comment]
[(0) PR Implemented feedback publishing by administrators](https://trello.com/c/luzaK7tm/56-0-pr-implemented-feedback-publishing-by-administrators)

**Pull request merged**